.coveragerc

```diff
@@ -1271,6 +1271,7 @@ omit =
     homeassistant/components/zwave/util.py
     homeassistant/components/zwave_js/discovery.py
     homeassistant/components/zwave_js/sensor.py
+    homeassistant/components/zyxelt50/*
```

Comment:
Config flow must have tests (and you have) so it can't be excluded here, add only the relevant files to be excluded